### PR TITLE
DTSPO-4987 ITHC: updated neuvector chart to use nodeaffinity

### PR DESCRIPTION
**Pull request by mokainos**

### CODE REVIEW

File: k8s/namespaces/neuvector/patches/ithc/cluster-00/neuvector.yaml [hunk 1]
```diff
@@ -67,3 +67,16 @@ spec:
             traefik.ingress.kubernetes.io/router.tls: "true"
           host: sds-neuvector00.ithc.platform.hmcts.net
           enabled: true
+      cve:
+        affinity:
+          podAntiAffinity:
```
Comment (timja) [inline k8s/namespaces/neuvector/patches/ithc/cluster-00/neuvector.yaml:72]: this is pod anti affinity, we want node anti affinity and it should be against the system node pool, see:
https://github.com/hmcts/aks-sds-deploy/blob/b1caab9a0777e4bad183414bee3a5b21c308eac3/kubernetes/charts/fluxcd/flux-values.yaml

also once it's working it should be moved to our helm chart (chart-neuvector) where we set some defaults on top of the upstream chart

File: k8s/namespaces/neuvector/patches/ithc/cluster-00/neuvector.yaml [hunk 2]
```diff
@@ -61,13 +87,11 @@ spec:
               Syslog_In_Json: false
               Cluster_Name: sds-ithc00
       manager:
-        ingress:
-          annotations:
-            kubernetes.io/ingress.class: traefik
-            traefik.ingress.kubernetes.io/router.tls: "true"
```
Comment (timja) [inline k8s/namespaces/neuvector/patches/ithc/cluster-00/neuvector.yaml:67]: Why did you remove these? Is it duplicated with another level?

Comment (mokainos) [reply k8s/namespaces/neuvector/patches/ithc/cluster-00/neuvector.yaml:67]: Sorry which one are u referring to?

Comment (timja) [reply k8s/namespaces/neuvector/patches/ithc/cluster-00/neuvector.yaml:67]: these 4 lines

**Review by timja — APPROVED** [review]
